envs/development/conda/fedbiomed-node.yaml

```diff
@@ -27,7 +27,7 @@ dependencies:
   - git ~=2.40.1
   - packaging ~=23.1
   # these two have to be aligned
-  - cryptography ~=40.0.0
```

Comment:
Thank you @rtaiello !

We should probably also update the files `envs/vpn/conda/fedbiomed-node.yaml` and  `envs/vpn/conda/fedbiomed-researcher.yaml` to keep coherent configuration for the docker/VPN packaging, shouldn't we ?

Thanks !

Reply:
Fixed it!